src/DataLib/fendl32B_retrofit/fendl3_gendf.py

```diff
@@ -0,0 +1,96 @@
+# Import packages
+import gendf_tools as GENDFtk
+import pandas as pd
+import subprocess
+from logging_config import logger
+import groupr_tools as GRPRtk
+
+def main():
```

Comment:
This main method is doing too many things and I don't know why.  You probably need some more modularity, with each function having a docstring to clarify what it's purpose is